### ISSUE TITLE
Bank avatar account is hard-coded

### ISSUE THREAD

**Issue by appurist** [InWorldz/halcyon#333]
As determined in #332, the avatar currency account is hard-coded:
```
public static UUID CURRENCY_ACCOUNT_ID = new UUID("efbfe4e6-95c2-4af3-9d27-25a35c2fd575");
```
This should be configurable via a `halcyon.ini` option.

There is an existing known value for this, named `EconomyBaseAccount` in the `[Economy]` section, which could be used for this. This option was only supported by certain development/example modules, specifically the SampleCurrencyModule, but I don't see any reason to create a new setting for this.

So this issue is in response to #332, and requests that the `CURRENCY_ACCOUNT_ID` be configurable via `EconomyBaseAccount` in the `[Economy]` section of the `halcyon.ini` file.

**Comment by appurist** [comment]
The `halcyon.ini` files have a historical `EconomyBaseAccount` setting which is not currently used because the SampleMoneyModule is not used.  If I commit a change that matches the historical name from that SampleMoneyModule, suddenly InWorldz economy settings would be wrong on many or all regions.

So since this is a new setting, I'm going to actually implement it as `CurrencyAccount` in the `[Economy]` section.

**Comment by appurist** [comment]
Okay I just implemented this right away since it was a quick change. Resolved in PR #334.
